Add sensors to onedrive for business (#43568)
home-assistant/home-assistant.github.io: source/_integrations/onedrive_for_business.markdown

- **Redirect URI**: Type: `Web`, URL: `https://my.home-assistant.io/redirect/oauth`

## Sensors

The integration provides the following sensors, which are updated every 5 minutes:

- **Total available storage**: The total size of your drive (disabled by default)
- **Used storage**: The amount of storage you have used up
- **Remaining storage**: The amount of storage that is left in your drive

> - **Drive state**: Calculated state of your drive, based on the storage left. Possible values: `Normal`, `Nearing limit`, `Critical`, `Exceeded`

Reviewer: ```suggestion
- **Total available storage**: The total size of your drive (disabled by default).
- **Used storage**: The amount of storage you have used up.
- **Remaining storage**: The amount of storage that is left in your drive.
- **Drive state**: Calculated state of your drive, based on the storage left. Possible values: `Normal`, `Nearing limit`, `Critical`, `Exceeded`.
```

{% note %}

> A drive that is in **Drive state** `Exceeded` will be automatically frozen (meaning you can't upload any more backups & files), until you clear up enough storage.

Reviewer: ```suggestion
A drive in **Drive state** `Exceeded` will be automatically frozen (meaning you can't upload any more backups & files) until you free up enough storage.
```
